test/e2e_node/util.go

```diff
@@ -106,6 +106,21 @@ func getNodeSummary() (*stats.Summary, error) {
 	if err != nil {
 		return nil, fmt.Errorf("failed to parse /stats/summary to go struct: %+v", resp)
 	}
+
+	framework.Logf("ruiwen-zhao: /stats/suammry: %+v", summary)
+	for _, p := range summary.Pods {
+		framework.Logf("ruiwen-zhao: pod: %+v", p.PodRef.Name)
```

Comment:
you should filter out the pods that are not in test here, you are panicing on pods that are not related to the test

Reply:
Current change will check the nil-ness of every field before dereferencing the pointer, so we should be fine, no?

The most recent test run seems to be ok: https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/109371/pull-kubernetes-node-e2e-containerd/1516927217286778880/